nuxeo-features/nuxeo-automation/nuxeo-automation-test/src/test/java/org/nuxeo/ecm/automation/server/test/TestBulkDownloadOperation.java

```diff
@@ -98,6 +112,9 @@ public void iCanBulkDownloadFiles() throws IOException, OperationException, Inte
             docs.add(doc);
         }
 
+        TransactionHelper.commitOrRollbackTransaction();
+        TransactionHelper.startTransaction();
```

Comment:
`TransactionalFeature#nextTransaction` or `CoreFeature#waitForAsyncCompletion` would be better here as there's async works.